MacrobenchmarkSample/app/src/main/java/com/example/macrobenchmark/target/activity/ComposeActivity.kt

```diff
@@ -45,25 +64,34 @@ class ComposeActivity : ComponentActivity() {
                         testTagsAsResourceId = true
                     }
                 ) {
-                    LazyColumn(
-                        modifier = Modifier
-                            .fillMaxSize()
-                            // Thanks to [SemanticsPropertyReceiver.testTagsAsResourceId], 
-                            // this [Modifier.testTag] will be propagated to resource-id 
-                            // and can be accessed from benchmarks.
-                            .testTag("myLazyColumn")
-                    ) {
-                        items(data, key = { it.contents }) { item ->
-                            EntryRow(entry = item,
-                                Modifier
-                                    .padding(8.dp)
-                                    .clickable {
-                                        ClickTrace.onClickPerformed()
-                                        AlertDialog
-                                            .Builder(this@ComposeActivity)
-                                            .setMessage("Item clicked")
-                                            .show()
-                                    })
+                    // Thanks to [SemanticsPropertyReceiver.testTagsAsResourceId],
+                    // [Modifier.testTag]s will be propagated to resource-id
+                    // and can be accessed from benchmarks.
+                    Column(modifier = Modifier.fillMaxWidth()) {
+                        TextField(
```

Comment:
you think it's worth implementing the mutable state here to make it more realistic?

Reply:
Added. Thanks!